WORKSPACE

```diff
@@ -1,61 +1,34 @@
 workspace(name = "io_tweag_rules_nixpkgs")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//nixpkgs:repositories.bzl", "rules_nixpkgs_dependencies")
 
-rules_nixpkgs_dependencies()
-
-load(
-    "//nixpkgs:nixpkgs.bzl",
-    "nixpkgs_cc_configure",
-    "nixpkgs_git_repository",
-    "nixpkgs_java_configure",
-    "nixpkgs_local_repository",
-    "nixpkgs_package",
-    "nixpkgs_python_configure",
-    "nixpkgs_sh_posix_configure",
-)
-
 # For documentation
 
-http_archive(
-    name = "io_bazel_stardoc",
-    sha256 = "6d07d18c15abb0f6d393adbd6075cd661a2219faab56a9517741f0fc755f6f3c",
-    strip_prefix = "stardoc-0.4.0",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/stardoc/archive/0.4.0.tar.gz",
-        "https://github.com/bazelbuild/stardoc/archive/0.4.0.tar.gz",
-    ],
-)
+load("@io_tweag_rules_nixpkgs//nixpkgs:repositories.bzl", "rules_nixpkgs_dependencies")
```

Comment:
you load the `rules_nixpkgs_dependencies` twice, first on line 3 and here again.